rdr_service/offline/sync_consent_files.py

```diff
@@ -42,6 +45,117 @@ class ParticipantPairingInfo:
     site_name: str
 
 
+@dataclass
```

Comment:
In case I wasn't seeing it:  what features of @dataclass are required/wanted here?

Reply:
Dataclass isn't exactly required, but just makes it easier to put information into an object. It takes the field definitions and makes them into instance variables for the objects. It also automatically provides a constructor so it's easy to instantiate the objects.